templates/requiredField.interface.hbs

```diff
@@ -8,5 +8,5 @@
     </select>
     <span class="icon fa fa-chevron-down"></span>
   </label>
-  <input name="value" class="form-control hidden" type="text" placeholder="Value"/>
+  <input name="value" class="form-control hidden" type="text" placeholder="Value" data-toggle="tooltip" data-placement="top" title="You can also use \{{&nbsp;user.[*]&nbsp;}} to reference user data, e.g. \{{&nbsp;user.[Email]&nbsp;}},<br />\{{&nbsp;user.[First&nbsp;name]&nbsp;}}"/>
```

Comment:
I remembered https://developers.google.com/style/clause-order where we can also achieve the same with less words. I usually try to stick by this where possible.

```suggestion
  <input name="value" class="form-control hidden" type="text" placeholder="Value" data-toggle="tooltip" data-placement="top" title="To reference user data, use \{{&nbsp;user.[*]&nbsp;}} e.g. \{{&nbsp;user.[Email]&nbsp;}},<br />\{{&nbsp;user.[First&nbsp;name]&nbsp;}}"/>
```

Reply:
@tonytlwu done